src/FsAutoComplete.Core/Commands.fs

```diff
@@ -746,6 +785,40 @@ type Commands (serialize : Serializer) =
             return [ Response.compile serialize (errors,code)]
     }
 
+    member __.BuildSymbolCacheForProject proj =
+        match state.Projects.TryFind proj with
+        | None -> ()
+        | Some pr ->
+            match pr.Response with
+            | None -> ()
+            | Some r ->
+                SymbolCache.buildProjectCache serialize r.Options
+                |> Async.Start
+
+    member __.BuildBackgroundSymbolsCache () =
+        async {
+            let start = DateTime.Now
+            SymbolCache.startCache state.WorkspaceRoot
+            do! Async.Sleep 100
```

Comment:
What we are waiting for here? Is 100ms a reliable value?

Reply:
For the Suave server in the symbol cache process. In my experience with Suave 100ms seems pretty reliable, tho "correct" way would be reading from symbol cache process standard output and waiting for the "listener started in" message from Suave.